shipyard-controller/handler/fake/configurationservice.go

```diff
@@ -0,0 +1,180 @@
+package fake
+
+import (
+	"encoding/json"
+	keptnapimodels "github.com/keptn/go-utils/pkg/api/models"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+)
+
+type MockConfigurationService struct {
```

Comment:
since this is now in its owm `fake` package I think we should rename it to `ConfigurationService` - 
imho `fake.ConfigurationService` looks cleaner than `fake.MockConfigurationService`